### PR TITLE
Enhance simple switch docs

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

File: docs/simple_switch.md [hunk 1]
```diff
@@ -36,6 +122,7 @@ header_type intrinsic_metadata_t {
         egress_rid : 16;
         resubmit_flag : 8;
         recirculate_flag : 8;
+        clone_spec : 32;   // TBD - should this be added in order for P4_14 clone to work?
```
Comment (antoninbas) [inline docs/simple_switch.md]: `clone_spec` must come from standard metadata, `intrinsic_metadata.clone_spec` is not used anywhere AFAIK

File: docs/simple_switch.md [hunk 2]
```diff
@@ -53,21 +140,54 @@ not access directly.
...
+feature. As a reminder, `resubmit` needs to be called in the ingress
+pipeline. See the "after-ingress pseudocode" for relative priority of
+this vs. other possible packet operations at end of ingress.
 - `recirculate_flag`: should not be accessed directly. It is set by the
 `recirculate` action primitive and is required for the recirculate feature. As a
-remainder, `recirculate` needs to be called from the egress pipeline.
+reminder, `recirculate` needs to be called from the egress pipeline.
+See the "after-egress pseudocode" for the relative priority of this
+vs. other possible packet operations at the end of egress processing.
+- `clone_spec`: should not be accessed directly. It is set by the
```
Comment (antoninbas) [inline docs/simple_switch.md]: Needs to be moved to `standard_metadata`

File: docs/simple_switch.md (hunk 1 above)
Comment (jafingerhut) [reply docs/simple_switch.md]: I removed this line.

File: docs/simple_switch.md (hunk 2 above)
Comment (jafingerhut) [reply docs/simple_switch.md]: I moved the description of clone_spec to the earlier list of standard_metadata fields.

File: docs/simple_switch.md [hunk 3]
```diff
@@ -94,6 +214,8 @@ queue.
 - `qid`: when there are multiple queues servicing each egress port (e.g. when
 priority queueing is enabled), each queue is assigned a fixed unique id, which
 is written to this field. Otherwise, this field is set to 0.
+TBD: `qid` is not currently part of type `standard_metadata_t` in v1model.
+Perhaps it should be added?
```
Comment (jafingerhut) [inline docs/simple_switch.md]: @antoninbas Should I maybe create a separate issue to add the `qid` field to v1model.p4 include file?

**Pull request merged**